library/Mockery/Generator/Parameter.php

```diff
@@ -56,7 +62,13 @@ public function getTypeHintAsString()
         }
 
         if (preg_match('/^Parameter #[0-9]+ \[ \<(required|optional)\> (?<typehint>\S+ )?.*\$' . $this->rfp->getName() . ' .*\]$/', $this->rfp->__toString(), $typehintMatch)) {
+
             if (!empty($typehintMatch['typehint'])) {
+
+                if ($this->isNamespacedClass($typehintMatch['typehint'])) {
```

Comment:
I think some kind of a check needs to be in place here, for the case when `typehint` is not a class name, no? For example `callable`.